warn/warn.go

```diff
@@ -960,6 +960,83 @@ func nativePackageWarning(f *build.File, fix bool) []*Finding {
 	return findings
 }
 
+// findEmptyReturns searches for return statements without a value and returns whether
+// the current list of statements terminates (either by a return or fail() statements on the current
+// level in all subranches.
```

Comment:
I found it hard to guess what the function does based on its name. Do you have an idea for a better name?

The comment should say what `callback` is for.

Reply:
Renamed to `findReturnsWithoutValue`, updated the comment.